scripts/load_data.sh

```diff
@@ -0,0 +1,144 @@
+#!/bin/sh
+set -e
+
+apk -q --no-cache add curl jq
+
+pip -q install s3cmd
+
+for file in NA12878.bam NA12878_20k_b37.bam; do
+    curl -s -L -o $file "https://github.com/ga4gh/htsget-refserver/raw/main/data/gcp/gatk-test-data/wgs_bam/$file"
+
+    yes | /shared/crypt4gh encrypt -p /shared/c4gh.pub.pem -f $file
+    ENC_SHA=$(sha256sum "$file.c4gh" | cut -d' ' -f 1)
+    ENC_MD5=$(md5sum "$file.c4gh" | cut -d' ' -f 1)
+    s3cmd -q -c /shared/s3cfg put "$file.c4gh" s3://dummy_gdi.eu/"$file.c4gh"
+
+    ## get correlation id from uload message
+    CORRID=$(
+        curl -s -X POST \
+            -H "content-type:application/json" \
+            -u test:test http://rabbitmq:15672/api/queues/gdi/inbox/get \
+            -d '{"count":1,"encoding":"auto","ackmode":"ack_requeue_false"}' | jq -r .[0].properties.correlation_id
```

Comment:
```suggestion
            -d '{"count":1,"encoding":"auto","ackmode":"ack_requeue_false"}' | jq -r '.[0].properties.correlation_id'
```


Quote the `jq` expression.